src/Foundation/Commerce/code/App_Config/Include/Foundation/Wooli/Wooli.Foundation.Commerce.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename file to "Foundation.Commerce.config". Because all our configs without Wooli. prefix